README.md

```diff
@@ -1,2 +1,12 @@
 # common_interfaces
 A set of packages which contain common interface files (.msg and .srv).
+
+
+## Purpose
+
+Isolating the messages to communicate between stacks in a shared dependency allows nodes in dependent stacks to communicate without requiring dependencies upon each other.
+This repository has been designed to contain the most common messages used between multiple packages to provide a shared dependency which will eliminate a problematic circular dependency.
```

Comment:
@tfoote this statement is a bit too verbose about something that applies to any dependency relationship at any level, but OK.